app/templates/views/home.html

```diff
@@ -220,7 +220,29 @@ <h3>Read articles covering esoterica, philosophy, art and more…</h3>
           <p>Download the latest issue</p>
         </div>
         <div class="col-xs-12 col-sm-6 col-md-4">
-          <img src="/static/images/home-image-magazine.png" class="img-fluid">
+          <a data-target="#modalIMG" data-toggle="modal" href="#" class="color-gray-darker td-hover-none">
+            <img class="card-img-top card_img_size" src="/static/images/home-image-magazine.png">
+            <div class="card-body">
+              <a data-target="#modalIMG" data-toggle="modal" href="#">View Image</a>
```

Comment:
Can you remove `View Image` link as I don't think its necessary.

Reply:
Done

Reply:
Done